website/scripts/neo4j-wdc-2.js

```diff
@@ -1,328 +1,354 @@
 var config = {};
 
 function executeNeoQueries(config, username, password, mode, fnResults, fnReason) {
-    return new Promise(function (resolve, reject) {
-        if ((config.url) && (username) && (password)) {
-            var driver = neo4j.v1.driver(config.url, neo4j.v1.auth.basic(username, password));
-            var session = driver.session();
-            var p = [];
-            var resultsTable = [];
-            config.cypherQueries.forEach(function (query, i) {
-                if (!(query.trim() === "")) {
-                    if (mode === 0) {
-                        // Explain Query
-                        query = 'Explain ' + query;
-                    }
-                    if (mode === 1) {
-                        // inspect schema query
-                        var inspectRows = parseInt(config.inspectRows, 10);
-                        var queryWords = query.split(/[\s\n]+/);
-                        if (isNaN(inspectRows) || inspectRows == 0) inspectRows = 1000;
-                        if (queryWords.length > 2 && queryWords[queryWords.length - 2].toLowerCase() === "limit") {
-                            var limit = Number(queryWords[queryWords.length - 1]);
-                            if (!isNaN(limit) && limit > inspectRows) {
-                                queryWords[queryWords.length - 1] = inspectRows.toString();
-                                query = queryWords.join(" ");
-                            }
-                        } else {
-                            query = query + " limit " + inspectRows;
-                        }
-                    }
-                    resultsTable.push(config.tableNames[i]);
-                    p.push(session.run(query));
-                }
-            });
-            if (p.length > 0) {
-                Promise.all(p).then(function (results) {
-                    session.close();
-                    var resultsData = fnResults(mode, results);
-                    resolve({
-                        resultsTable: resultsTable,
-                        resultsData: resultsData
-                    });
-                }, function (reason) {
-                    session.close();
-                    fnReason(mode, reason);
-                    reject();
-                });
-            } else {
-                session.close();
-                resolve();
-            }
-        }
-    });
+	return new Promise(function(resolve, reject) {
```

Comment:
as far as I can see there is not a single content change in this file just reformatting 
please don't do that.
only submit real code changes.